radioactivedecay/inventory.py

```diff
@@ -293,58 +620,67 @@ def __len__(self) -> int:
 
         return len(self.contents)
 
-    def add(self, add_contents: Dict[Union[str, Radionuclide], float]) -> None:
+    def add(
+        self,
+        add_contents: Dict[Union[str, Radionuclide], float],
+        input_type: str = "activity",
+    ) -> None:
         """
-        Adds a dictionary of radionuclides and associated activities to the inventory.
+        Adds a dictionary of radionuclides and associated numbers to the inventory.
 
         Parameters
         ----------
         add_contents : dict
             Dictionary containing radionuclide strings or Radionuclide objects as keys and
             activities as values which are added to the Inventory.
+        input_type : str, optional
+            Type of input, default is activities.
```

Comment:
I like how you did it for the `__init__` constructor where you state all acceptable strings. Explicit is good!